ugo_collector/RatePlan_Created/RatePlan_Created_1_firstAnalysis.ipynb

```diff
@@ -0,0 +1,2472 @@
B8fy56d37F4wYdEXTjHs117pzvvu4WVuLhY1q79kj/+OMLgwW9YHT568WIUXl5eODs7pzlOrty5Abh65fIjFy9ERERERETk0ah4Iakym1NfKRAREUn//oNs2g8c2M/MmVNs2u+3auLenq7d+1r1lS1Xmdkz3mPTxrU0aNSSXL65MZvNmEy2Kzzuio+PtymsdO7cjRYtWqV5T1osz/8Iqz5EREREREQkc6h4Iam6eDEKgFy5fK3a3d3dCQsrZhN/6ZLta049PT2JiYmxab9x486KC1e3+7/JpEr1OuzZtY2TJ/4kl29udm7fzLzZU6xifvhhv+XPzs4uTJ8+G7PZzOXLl/nkk0V88skiQkJCqVOnniXOz8+fs2fPkJCQkObqi6uXLwPg45PrvjmKiIiIiIjI46fihaRq9+4dGAwGSpUq/dBjFCwYzKlTJ2zaz/z9F0ajkTx58t33fvP/r7IwGO6cK1s6sgIjx0xJM97OzmgprBQtCmXLluPZZ9sybdokKlWqgouLCwBly5bnxx9/YPfuHdSpU99mnKTERH7/bT958xfQlhEREREREZFsQG8bERsbN65n9+6d1K5dF3//gIcep3r1mpw4cZxjx/60tCUn3+aH3d9TPDwC5/8vJqRl146tGAxGCgWHAODm7kFQcIjV1/24uLjSp09/rl69ysqVn1vamzZtiZeXN7NnzyQ6Otrmvs8/WcCN+HiatmqXgacVERERERGRx0UrL55iKSkmDh06CEBSUiIXLpxn587v2bHje0qWjGDIkOGPNH7Tpi1YvfoLhg0bSp8+/fDy8mLBooVEX79Gv4GvW+L27NrGgZ/3Uqp0OXLm9CE+Pp7dO79j/097aNy0NT65cj90DvXqNWTZsk/47LMltG7dFhcXVzw8PBgzZgKvvfYyPXt2oWPHLgQHFyYmJoaNG9exY8f31G/SgopVaz7S84uIiIiIiEjmUPEiA27dSmLr4jezPIfMkpBwkz59ugPg5OSEt3dOihQJZdSocdSqVRej8dEW5jg6OjJ16iw+/HAqkya9S1JSEvkLFmLosDEEFfpn1YRvbn/i4mJYvnQB8XFx5MiRg8B8BejV52Wq1qj7SDkYjUZeeKEfQ4YM5IsvPuO5554H7hw8umjRMpYsWciyZUu4evUKTk7OFC1ajJdee5PS5dL3RhURERERERF5/AzmtF4r8T/s6tV4TKbUHzsq6jT+/gWecEZPB3t7I3+eupCpY4YGBZCcnPYbSDLK3t7IkbOZm2PRwIzlqJ9BEXmcfH3dqdJvdKaOuWvmm1y+HPfgwP8xvr7uVF/8aKsU77W969in8nMUEREBMBoN+Pi4pd3/BHMREREREREREckwFS9EREREREREJFtT8UJEREREREREsjUVL0REREREREQkW1PxQkRERERERESyNRUvRERERERERCRbU/FCRERERERERLI1FS9EREREREREJFtT8UJEREREREREsjX7rE5ARERE5N/cPB1xdnDItPESkpKIj0nMtPFERETkyVLxIgPcPBxxdsy8/5B6GAmJScTHZvw/vjZsWMu4caNYuPBTQkJCGTt2JDt2bGPjxm1Wcb/8sp/XXnuZgIC8TJ48A2/vnJa+gwd/pW/f53F2duGrrzbi4uKS6lz79v3A3LmzOX78GC4uLlSvXpO+fV/C29vTEnPk8G+8O/qNVO8fP2k2efLmu+/zzJ31ATu3b7Fc29nZ4eOTi4iISHr06E1g4D/33332e3l5eRMcHELXrt0pU6bcfecSEZEnz9nBgYgxIzNtvF9GjCQeFS9ERET+q1S8yABnRweq9BudpTnsmvnmY/uPrz17djJixGsUKRLKxIlTcXd3t+pfv34NAAkJN9m6dRNNm7awGWP//p8YPHgg1arVpFevvly5cpnZs6dz6tQJ5sxZYBPfrmN3QouGW7Xl8vVLV75OTs7MmDGL5GQzKSnJnDlzmgUL5jJwYF8WL/4MV1c3q/gRI0YRGJgfMHPlymWWL1/Kq68OYNas+RQtWjxdc4qIiIiIiMiTp+KFALBlyyZGj36TyMiyjBs3CScnJ6v+hIQEtm7dTIkSpfjrr1OsX78m1eLFrFnTKFQomHfeGY/ReOdIlVy5cvHyy/3ZsmUT+QuXtIr3D8hL4ZCwh8rZaDQSHl6S5GQTAKVKlcbd3ZPhw4dw8OBvVKxY2So+OLgwISGhluvIyLI0aVKXbdu2qnghIiIiIiKSjenATmHt2i8ZNWo4VapUY8KEyTaFC4DvvtvMzZs3aNGiFXXrNuDgwV/5++/TVjGXL1/iyJHDNGjQ2FK4AChXriK+vrnZtm3rY38WV1dXAJKTbz8w1snJGaPRmK5YERERERERyToqXjzlli37hAkTxtCwYRPeeeddcuTIkWrc+vVrcHFxpWbNOjRp0tzSdq+TJ08AUKhQYZv7CxUqzIkTx23aF82bQfdnm9GnR1s+mDiSUyePZSj/5ORkkpOTSUxM5Nixo8ybNxsfn1xERpa1iU1JMVnio6IuMHnyRABq1aqXoTlFRERERETkydK2kadYfHw8M2dOoWzZ8rzxxlsYDIZU486ePcOvvx6gWbOWODk5ERZWlODgEDZuXE/v3i9iZ2cHQExMDAAeHh42Y3h4eHD06B+WaxdnV+o3akHRYiVwdXPn/LkzrF/zBWNHDuGNt94luPCDt5LcvHmDqlXLW7UFBOTh/fen4+LiahPfs2cXq2snJydGjBhFeHiJB84lIiIiIiIiWUfFi6eYs7MzRYsWZ//+n9iwYa1lRcW/3V1h0bhxM0tbkybNmDbtA374YTdVqlT71x2pF0HuLY4UCAqmQFCw5To0LJzSZSowfEg/VixfzGvDxwFgMqVgNluPcXdLipOTM7NmzSUlxURKiomoqPN8+unHvPrqAGbM+Ih8+fJbzf/WW2PIn/9O2/Xr19iwYR1jx47EwcGBGjVq3+eTEhERERERkaykbSNPMTs7OyZOnEJERCQTJoxhw4a1NjEpK
```

Comment:
I think plot will be better readable with legend on the right plt.legend(loc='upper right')